Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/classic/MetaClassicUnitFragment.kt

```diff
@@ -0,0 +1,104 @@
+package com.taboola.kotlin.examples.screens.classic
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.facebook.ads.NativeAdLayout
+import com.taboola.android.MetaPlacementProperties
+import com.taboola.android.TBLPublisherInfo
+import com.taboola.android.Taboola
+import com.taboola.android.annotations.TBL_PLACEMENT_TYPE
+import com.taboola.android.listeners.TBLClassicListener
+import com.taboola.kotlin.examples.MetaConst
+import com.taboola.kotlin.examples.PlacementInfo
+import com.taboola.kotlin.examples.R
+
+class MetaClassicUnitFragment : Fragment() {
+
+    private var rootView: View? = null
+    private var adContainerTop: NativeAdLayout? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        Taboola.init(TBLPublisherInfo(MetaConst.META_PUBLISHER_NAME))
+        rootView = inflater.inflate(R.layout.fragment_meta_ad_classic_unit, null)
+        adContainerTop = rootView!!.findViewById<NativeAdLayout>(R.id.native_ad_container_top)
+        Taboola.setGlobalExtraProperties(object : HashMap<String?, String?>() {
```

Comment:
@voltfmanSlava - Any reason why you didn't use [hashMapOf](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.collections/hash-map-of.html) here?